library/src/main/java/ch/decent/sdk/api/ValidationApi.kt

```diff
@@ -1,11 +1,24 @@
+@file:Suppress("TooManyFunctions", "LongParameterList")
```

Comment:
we repeat this rule a lot. How about setting up linter so that this is not a problem?

Reply:
Agree on this.

Reply:
these rules are ignored only in `api` package and `Utils` class, where we break these rules by intention, but they should definitely apply for the rest of the code